### PR TITLE
Improve player color selection

### PR DESCRIPTION
**Pull request by vsrisuknimit**
- Color boxes instead of radio buttons for picking colors.
- The entire player box changes color to match, making it easier to tell who is playing what color.
- Hiding TR boost and Beginner Corp option under Beginner Options

image

### CODE REVIEW

**Comment by vsrisuknimit** [comment]
Still has to refactor the color classes as @chosta suggested. More commit coming.

**Comment by vsrisuknimit** [comment]
@chosta . I finished moving classes to common.less. Please check again. I deleted the last PR after my git rebase added a thousand commits.

**Comment by chosta** [comment]
Looks good to me. I like the name change to 'translucent', too.

**Comment by vsrisuknimit** [comment]
Example: 

image

image

**Comment by vsrisuknimit** [comment]
image

What it looks like top and bottom together. Tagging @alrusdi for review. Good to go if you think it is good.

**Comment by kberg** [comment]
I'm a little worried about this being particularly unfriendly to people with color blindness issues. Having the names of the colors would go a long way.

**Comment by vsrisuknimit** [comment]
@kberg I can do something like this but it is unclear how that would help. Users will still have to deal with the problem in the main games of cubes being the same color, even if they can tell the colors apart due to labels on the selection page.

We need to a setting for users to pick their own color palette.

image

**Comment by vsrisuknimit** [comment]
Labels with colorblind simulation.

image

**Comment by kberg** [comment]
Oh I love it! You're the best.

File: src/components/CreateGameForm.ts [hunk 1]
```diff
@@ -361,7 +361,7 @@ export const CreateGameForm = Vue.component("create-game-form", {
 
                     <div class="create-game-page-container">
                         <div class="create-game-page-column" v-if="! isSoloModePage">
-                            <h4 v-i18n># of Players</h4>
+                            <h4 v-i18n>№ of Players</h4>
```
Comment (alrusdi) [inline src/components/CreateGameForm.ts:364]: Please replace original string "Players count" with proposed "№ of Players" in all translation files

**Review by alrusdi — APPROVED** [review]

File: src/components/CreateGameForm.ts (hunk 1 above)
Comment (vsrisuknimit) [reply src/components/CreateGameForm.ts:364]: All set. I did not change the translation though.

**Pull request merged**